packages/docusaurus-theme-classic/src/theme/BlogPostItem/Header/Author/index.tsx

```diff
@@ -5,11 +5,70 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import type {SVGProps} from 'react';
 import React from 'react';
 import clsx from 'clsx';
 import Link, {type Props as LinkProps} from '@docusaurus/Link';
 
 import type {Props} from '@theme/BlogPostItem/Header/Author';
+import styles from './styles.module.css';
+
+type SocialProps = SVGProps<SVGSVGElement> & {
+  user: string;
+};
+
+function getUrl(user: string, baseUrl: string) {
+  return user.startsWith('https') ? user : `${baseUrl}${user}`;
+}
+
+function Twitter(props: SocialProps) {
+  const twitterUrl = getUrl(props.user, 'https://x.com/');
+  return (
+    // eslint-disable-next-line @docusaurus/no-html-links
+    <a
+      href={twitterUrl}
+      target="_blank"
+      rel="noreferrer noopener"
+      className={clsx(styles.socialIcon)}>
+      <svg
+        viewBox="0 0 256 209"
+        width="1.5em"
+        height="1.5em"
+        xmlns="http://www.w3.org/2000/svg"
+        className={clsx(styles.socialIcon)}
+        preserveAspectRatio="xMidYMid"
+        {...props}>
+        <path
+          d="M256 25.45c-9.42 4.177-19.542 7-30.166 8.27 10.845-6.5 19.172-16.793 23.093-29.057a105.183 105.183 0 0 1-33.351 12.745C205.995 7.201 192.346.822 177.239.822c-29.006 0-52.523 23.516-52.523 52.52 0 4.117.465 8.125 1.36 11.97-43.65-2.191-82.35-23.1-108.255-54.876-4.52 7.757-7.11 16.78-7.11 26.404 0 18.222 9.273 34.297 23.365 43.716a52.312 52.312 0 0 1-23.79-6.57c-.003.22-.003.44-.003.661 0 25.447 18.104 46.675 42.13 51.5a52.592 52.592 0 0 1-23.718.9c6.683 20.866 26.08 36.05 49.062 36.475-17.975 14.086-40.622 22.483-65.228 22.483-4.24 0-8.42-.249-12.529-.734 23.243 14.902 50.85 23.597 80.51 23.597 96.607 0 149.434-80.031 149.434-149.435 0-2.278-.05-4.543-.152-6.795A106.748 106.748 0 0 0 256 25.45"
+          fill="#55acee"
+        />
+      </svg>
+    </a>
```

Comment:
Where does this SVG come from, what's the license? can you put a comment to the source?

I propose to use https://svgl.app/ (MIT)